### ISSUE TITLE
update wikipedia's lightweight markup comparison page with asciidoctor

### ISSUE THREAD

**Issue by zaxebo1** [asciidoctor/asciidoctor#1303]
the wikipedia page
  https://en.wikipedia.org/wiki/Comparison_of_lightweight_markup_languages
  https://en.wikipedia.org/wiki/Comparison_of_document_markup_languages
compares the lightweight markup languages.
This page uses asciidoc (the python implementation) to compare the features. Asciidoctor(this ruby based implementation) implements lots of missing functionality in the asciidoc markup too. **This wikipedia page may be updated with feature comparson table in respect of asciidoctor features** , so that new users can compare the features and select asciidoctor. They should not get demotivated in selecting asciidoctor, when they compare asciidoc(python implementation) missing features, which are available in sphinx.

==
additionally here in this following table also "sphinx" appears, but asciidoctor is not compared there
https://en.wikipedia.org/wiki/Comparison_of_documentation_generators

**Comment by mojavelinux** [comment]
What I would prefer to see is the syntax that Asciidoctor supports be represented in the AsciiDoc column. Asciidoctor is now the leading implementation of AsciiDoc and in my eyes, what is in Asciidoctor is the official AsciiDoc now. In contrast, the syntax supported by AsciiDoc Python would be listed as "Legacy AsciiDoc" if listed at all.

**Comment by tlberglund** [comment]
Agreed, @mojavelinux. I think this needs to be the position of the project. I don't know actual user statistics, but there is no question about which has more innovation or active development.

**Comment by zaxebo1** [comment]
on the page https://en.wikipedia.org/wiki/Lightweight_markup_language , we can see that  markdown,markdown extra, multi markdown are separately listed simultaneously. So may be , asciidoc and asciidoctor both can co-exist on these pages' comparison tables

**Comment by zaxebo1** [comment]
additionally there is a wikipedia page for asciidoc https://en.wikipedia.org/wiki/AsciiDoc

but there is NO wikipedia page for asciidoctor

compare this to situation that - there is wikipedia page 
https://en.wikipedia.org/wiki/ReStructuredText
https://en.wikipedia.org/wiki/Sphinx_%28documentation_generator%29

**Comment by zaxebo1** [comment]
Or, a better proposal:
from nowonwards:
 "asciidoc" should be  the specification of this language.
  **"asciidoc classic" is the first and oldest implementation of this specification (in python)**
  "asciidoctor" is the more popular implementation of asciidoc specification.(in ruby)
  whenever we talk about asciidoc implementation, then by default we are talking of asciidoctor, unless otherwise stated

By this proposal we are differentiating the confusion between asciidoc specification and asciidoc(python implementation), by renaming asciidoc (python implementation) as "asciidoc classic"

**Comment by mojavelinux** [comment]
I'm still in favor of the terminology AsciiDoc Python and Asciidoctor, as these are the names of the implementations.

When we talk about the AsciiDoc standard, we should be talking about the language itself (and eventually UniDoc, see https://github.com/asciidoctor/asciidoctor/wiki/AsciiDoc-Specification-(aka-UniDoc)-Planning). How AsciiDoc is implemented is really a separate issue. Having said that, we can be free to talk about the language including features that are only implemented in Asciidoctor...because we are driving the language forward.

**Comment by mojavelinux** [comment]
cc: @graphitefriction

**Comment by lordofthejars** [comment]
> I'm still in favor of the terminology AsciiDoc Python and Asciidoctor, as these are the names of the implementations.

+1
